packages/api/src/controllers/taskController.js

```diff
@@ -110,6 +129,21 @@ const taskController = {
         );
       }
 
+      // update wage and don't always ask wage
+      const trx = await transaction.start(Model.knex());
```

Comment:
can look into other ways to query the table. Please check how it's been done in the other controllers.